tests/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    etc_hosts: passes path for etc_hosts fixture
```

Comment:
I had to add this to root's `pytest.ini` to remove the warnings.

Reply:
Moved, the other removed.